src/complete.cpp

```diff
@@ -450,7 +450,7 @@ void complete_add(const wchar_t *cmd, bool cmd_is_path, const wcstring &option,
                   const wchar_t *condition, const wchar_t *comp, const wchar_t *desc,
                   complete_flags_t flags) {
     assert(cmd && "Null command");
-    // option should be empty iff the option type is arguments only.
+    // option should be empty if the option type is arguments only.
```

Comment:
This is actually an "iff":

>: if and only if 

>two figures are congruent *iff* one can be placed over the other so that they coincide 

https://www.merriam-webster.com/dictionary/iff

Reply:
Ah, wasn't aware of that abbreviation. Reverted.